docs/build-transaction-construction.md

```diff
@@ -0,0 +1,208 @@
+---
+id: build-transaction-construction
+title: Transaction Construction and Signing
+sidebar_label: Transaction Construction
+---
+
+This page will discuss the transaction format in Polkadot and how to create, sign, and broadcast
+transactions. Like the other pages in this guide, this page demonstrates some of the available
+tools. Always refer to each tool's documentation when integrating.
+
+## Transaction Format
+
+Polkadot has some basic transaction information that is common to all transactions.
+
+- Address: The SS58-encoded address of the sending account.
+- Block Hash: The hash of the [checkpoint](build-protocol-info#transaction-mortality) block.
+- Block Number: The number of the checkpoint block.
+- Genesis Hash: The genesis hash of the chain.
+- Metadata: The SCALE-encoded metadata for the runtime when submitted.
```

Comment:
There's something not well documented in txwrapper/polkadotjs tools: overriding the type registry. E.g. when users have custom types in their chain.

- PolkadotJS tools has an open issue to add them
- `txwrapper` allows them, but it's not well documented yet (defaults to Kusama)

It's already a slightly advanced topic, so I'll let you decide whether we should mention that in the wiki or not. Just wanted to let you know.

Reply:
Since this is for the Polkadot Wiki, it doesn't have to be generic over all Substrate chains. We should mention how to handle Kusama/Polkadot though.